src/components/dialogs/EditPassiveElement.jsx

```diff
@@ -10,30 +10,30 @@ export function EditPassiveElement({
 }) {
   const networkElements = useContext(PassivesContext);
 
-  const elementSelected = networkElements.find(
+  const { name, id } = networkElements.find(
```

Comment:
are you sure you are not missing something here?

i did test the cahnges and i found this:

![image](https://github.com/jignaciovelazquez/grilla-react-1/assets/18018741/237d6749-dee5-4a95-bee8-b6783d91b670)